### PR TITLE
Add fix for max sell and redeem by rounding down values

### PR DESCRIPTION
**Pull request by pRoy24**
Round down to 14 precision points .
This prevents overflow and ensures that there are no precision issues while selling.

Fixes https://github.com/protofire/omen-exchange/issues/1676
https://github.com/protofire/omen-exchange/issues/1757

### CODE REVIEW

**Comment by Mi-Lan** [comment]
Works for me GJ! I tried it on this market on xDai where i originally had issue with one long position. http://localhost:3000/#/0x09ba10798a54f8830e3568a000e55981c493f043

**Review by Mi-Lan — APPROVED** [review]
Works for me GJ! I tried it on this market on xDai where i originally had issue with one long position. http://localhost:3000/#/0x09ba10798a54f8830e3568a000e55981c493f043

File: app/src/components/market/sections/market_sell/market_sell.tsx [hunk 1]
```diff
@@ -144,10 +145,12 @@ const MarketSellWrapper: React.FC<Props> = (props: Props) => {
       const holdingsOfOtherOutcomes = holdings.filter((item, index) => {
         return index !== outcomeIndex
       })
-
+      const roundingFactor = STANDARD_DECIMALS - 4
```
Comment (KadenZipfel) [inline app/src/components/market/sections/market_sell/market_sell.tsx:148]: It looks like `amountShares` isn't actually always 18 decimals, but instead tracks the amount of decimals of the collateral

**Review by KadenZipfel — CHANGES_REQUESTED** [review]

**Comment by KadenZipfel** [comment]
I've reverted back to the old rounding strategy, reason being that since we're working with `BigNumber`s, the decimals don't change during arithmetic, so it works well.

**Review by Mi-Lan — APPROVED** [review]
Works!

**Review by hexyls — APPROVED** [review]

**Review by gem1no — APPROVED** [review]
Clever way of handling this, works for me. Approved!

**Review by pimato — APPROVED** [review]
works!

**Pull request merged**